ex1/functions.py

```diff
@@ -65,23 +91,99 @@ def f():
 # compare against. The function should return a list with equal length as the
 # input list containing for each number in the original list either True or
 # False if the number was greater than or equal to n.
+def check_list(numbers, n):
+    """
+    checks for every number of a list if it is greater than or equal to the
+    given number n (True) or not (False)
+    Paramters
+    ----------
+    numbers: list
+        numbers to compare
+    n: integer, float_number
+        number to compare with
+    Returns
+    ----------
+    answer: list
+        true/false answers
+    """
+    i = 0  
+    answer = []
+    for number in numbers:
+        answer.append(number >= n)
+    return answer
```

Comment:
You can solve this and the next one with list comprehension. See #20 for an example.